include/net/buf.h

```diff
@@ -344,7 +344,151 @@ void net_buf_simple_add_le64(struct net_buf_simple *buf, uint64_t val);
 void net_buf_simple_add_be64(struct net_buf_simple *buf, uint64_t val);
 
 /**
- * @brief Push data to the beginning of the buffer.
+ * @brief Remove data from the end of the buffer.
+ *
+ * Removes data from the beginning of the buffer by modifying the buffer length.
```

Comment:
Aren't we removing data from **the end of the buffer** instead of beginning?

Reply:
Yes, I missed this when converting comments. Thanks